docs/howto/operate/move-hub.md

```diff
@@ -23,29 +23,67 @@ home directory storage!
 
 Primarily used with GKE right now.
 
-1. In the target NFS server, create a new ssh key-pair, with
+1. SSH into **both** the target server and source server and then enable the `ubuntu` user on **both** servers with `sudo su ubuntu`.
+   This will ensure that the keys we are about to create will be assigned to the correct user.
+2. In the target NFS server, create a new ssh key-pair, with
    `ssh-keygen -f nfs-transfer-key`
-
-2. Copy the public key `nfs-transfer-key.pub` to the source NFS
-   server's `/home/ubuntu/.ssh/authorized_keys`, so the target
-   NFS server can `scp` from the source NFS server.
-
-3. Copy the NFS home directories from the source NFS server to
+3. Append the public key `nfs-transfer-key.pub` to the source NFS
+   server's `/home/ubuntu/.ssh/authorized_keys` file, so the target
+   NFS server can open SSH connections to the source NFS server.
+4. Copy the NFS home directories from the source NFS server to
    the target NFS server, making sure that the NFS exports locations
    match up appopriately. For example, if the source NFS server has
    home directories for each hub stored in `/export/home-01/homes`,
    and the target NFS server also has hub home directories stored under
    `/export/home-01/homes`, you can `scp` the contents across with:
 
    ```bash
-   su ubuntu
    scp -p -r -i nfs-transfer-key ubuntu@nfs-source-server-public-IP:/export/home-01/homes/<hub-name> /export/home-01/homes/<hub-name>
    ```
 
    This makes sure the target is owned by the `ubuntu` user, which has
    uid `1000`. Our user pods run as uid `1000`, so this makes sure they
    can mount their home directories.
 
+As an alternative to `scp` you can use `rsync` as follows:
+
+```bash
+rsync -e 'ssh -i nfs-transfer-key' -rougvhP ubuntu@nfs-source-server-public-IP:/export/home-01/homes/<hub-name>/ /export/home-01/homes/<hub-name>/
+```
```

Comment:
Could it be the case the `-e 'ssh -i nfs-transfer-key'` is not actually needed if you ssh-add the key?

Reply:
I don't know, I'm not familiar enough with `ssh add`. What I found is that because I prefaced the command with `sudo`, `rsync` was looking at `root`'s ssh keys, so I was just specific in saying which key I wanted to use.